assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -262,19 +265,25 @@
 
 			setSide( event );
 
-			if ( ! isDroppingAllowed( event ) ) {
-				return;
-			}
+			elementor.browserImport
+				.createSession( event.originalEvent.dataTransfer.items, settings.getDropContainer() )
+				.then( async ( session ) => {
+					isDroppingAllowedState = isDroppingAllowed( event ) || await session.validate();
+
+					if ( ! isDroppingAllowedState ) {
+						return;
+					}
 
-			insertPlaceholder();
+					insertPlaceholder();
 
-			elementsCache.$element.addClass( settings.hasDraggingOnChildClass );
+					elementsCache.$element.addClass( settings.hasDraggingOnChildClass );
 
-			$( currentElement ).addClass( settings.currentElementClass );
+					$( currentElement ).addClass( settings.currentElementClass );
 
-			if ( 'function' === typeof settings.onDragEnter ) {
-				settings.onDragEnter.call( currentElement, currentSide, event, self );
-			}
+					if ( 'function' === typeof settings.onDragEnter ) {
+						settings.onDragEnter.call( currentElement, currentSide, event, self );
+					}
+				} );
```

Comment:
Use/Create a command.

Reply:
The purpose of this block of code is to validate that a given list of items can be imported using the `browser-import` module. In order to do that, we have to create a new `Session` and use the `validate` method. We can't use the current `browser-import` command since it's also creating the elements given to it, which is not the wanted case here - where we only want to validate that the files can be imported.
Do you suggest to create a new command only for the purpose of validating the given items?